dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/exceptions/i18n/ConcurrencyExceptionResource.nlsprops

```diff
@@ -39,13 +39,13 @@
 
```

Comment:
Add 2023 to copyright

Reply:
A nlsprops verification tool in the build does not currently allow the copyright to be updated.